common/lib/xmodule/xmodule/modulestore/tests/django_utils.py

```diff
@@ -8,7 +8,7 @@
 from contextlib import contextmanager
 from enum import Enum
 
-from courseware.field_overrides import OverrideFieldData  # pylint: disable=import-error
+from lms.djangoapps.courseware.field_overrides import OverrideFieldData  # pylint: disable=import-error
 from courseware.tests.factories import StaffFactory
```

Comment:
I changed these specific ones because `field_overrides` had a thread-local in it that was getting created twice (once in each namespace).

We didn't clean it all up a long time ago because we have some external code that's importing files directly from `courseware`, I believe. I would love to get rid of those so that we can stop changing `sys.path` in our django settings, though.